LT.DigitalOffice.Tests.sln

```diff
@@ -14,10 +14,25 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution
 		README.md = README.md
 	EndProjectSection
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "GUITestProject", "GUITestProject\GUITestProject.csproj", "{33E3D1CF-7CAB-435E-8B71-C51E7DAFDAE6}"
+EndProject
 Global
+	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|Any CPU = Debug|Any CPU
+		Release|Any CPU = Release|Any CPU
+	EndGlobalSection
+	GlobalSection(ProjectConfigurationPlatforms) = postSolution
+		{33E3D1CF-7CAB-435E-8B71-C51E7DAFDAE6}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{33E3D1CF-7CAB-435E-8B71-C51E7DAFDAE6}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{33E3D1CF-7CAB-435E-8B71-C51E7DAFDAE6}.Release|Any CPU.ActiveCfg = Release|Any CPU
+		{33E3D1CF-7CAB-435E-8B71-C51E7DAFDAE6}.Release|Any CPU.Build.0 = Release|Any CPU
```

Comment:
where do you get this Guid and what's going on here?

Reply:
It's just adding a new project to the solution.

If you want to understand in more detail how the sln works, you can read this https://xoofx.com/blog/2014/05/25/managing-multiple-platforms-in-visual/